pkg/admin.go

```diff
@@ -81,35 +81,47 @@ func (r *Registry) VendorClaim(vendorID string, orgID string, orgName string, or
 	if vendor == nil {
 		return nil, fmt.Errorf("vendor not found (id=%s)", vendorID)
 	}
-	certificates := vendor.GetActiveCertificates()
-	if len(certificates) == 0 {
-		return nil, fmt.Errorf("vendor has no active certificates (id = %s)", vendorID)
-	}
 
-	// If no keys are supplied, make there's a key in the crypto module for the organisation
-	if orgKeys == nil || len(orgKeys) == 0 {
-		logrus.Infof("No keys specified for organisation (id = %s). Keys will be generated or loaded from crypto module.", orgID)
+	// If no keys are supplied, make sure there's a key in the crypto module for the organisation
```

Comment:
What if the certificates are expired? Should we generate new keys? This feels like a explicit administration step, not a side effect of making a vendorClaim

Reply:
When certificates are expired but the keys haven't (according to the key rotation scheme in place), certificates should be reissued. When keys are expired by the key rotation scheme, new keys must be generated and new certificates issued.

That is not part of this bugfix however and we don't support it yet. We need to be able to update the registry for that.